### DISCUSSION TITLE
Set Up loop on RTL

### DISCUSSION THREAD

**Discussion by dubem2** [Ride-The-Lightning/RTL#1274] [LND]
I setup RTL on an lnd node running on mainnet, I’ve been able to configure the boltz feature and it works fine, but the loop out feature doesn’t work despite specifying the dir for loop.macaroon in RTL config file. I’d appreciate any tips on how to make it work. Below are screenshots of my RTL-Config.json file and RTL UI loop section(when i try to use the loop out feature)

<img width="1011" alt="Screen Shot 2023-08-11 at 1 10 53 AM" src="https://github.com/Ride-The-Lightning/RTL/assets/60652826/39454682-e0a6-450e-b907-7d7448e26e02">
<br/>
<br/>
<img width="1440" alt="Screen Shot 2023-08-11 at 1 12 50 AM" src="https://github.com/Ride-The-Lightning/RTL/assets/60652826/b4b0c383-52fb-4fa5-9867-e10ea470203b">

**Comment by ShahanaFarooqui** [comment]
Could you please share your RTL, lnd and loop versions as well?

**Reply by dubem2** [reply]
RTL version v0.14.0-beta 
Loop version 0.25.2-beta commit=v0.25.2-beta
Lnd version 0.16.4-beta commit=v0.16.4-beta
